lib/facilities/repo.ex

```diff
@@ -3,11 +3,13 @@ defmodule Facilities.Repo do
   Repo to get facilitiy information.
   """
 
+  alias MBTA.Api.Facilities
```

Comment:
Alias should go below `use`. We want to order include directives based on the level of their effect. `require`, `use`, `import`, `alias`. Alias is last because it has no effect other than giving a shorter name.